pkg/graph/schema/types/model_to_operations_matrix.graphql

```diff
@@ -0,0 +1,95 @@
+# ModelsToOperationMatrix Notes:
+# - There's purposefully no `Milestones` query right now, since there's currently no need to query milestones oustide the scope of a Category
+type ModelsToOperationMatrix {
+  categories: [Category!]! # Categories is at the top level because you _can_ have empty categories
```

Comment:
We need a way to retrieve un-categorized milestones as part of this. 

We may need to represent an empty category in the response, just so we can return milestones that have a null category.

Reply:
Yeah -- currently this deesign (of just having a `categories` and no `milestones` property would demand that we have some sort of `"Uncategorized"` category returned)

If we don't go that route, we should proabably just add a `milestones` property here, which is also a pretty simple resolver!